### PR TITLE
:sparkles: add maven index

### LINKED ISSUE #53: Add maven index file
We need to add a default maven index file in here so that analyzer can use it by default when no other index is provided.

### PR DESCRIPTION
**Pull request by pranavgaikwad**
Closes #53

### CODE REVIEW

**Comment by pranavgaikwad** [comment]
I am able to build the project locally, but somehow building it in the Dockerfile is taking awful lot of time on my machine. I tested the maven index part in isolation in a different Dockerfile and it works as expected.

File: Dockerfile [hunk 1]
```diff
@@ -28,4 +33,5 @@ COPY --from=jdtls-download /jdtls /jdtls/
 COPY --from=addon-build /root/.m2/repository/io/konveyor/tackle/java-analyzer-bundle.core/1.0.0-SNAPSHOT/java-analyzer-bundle.core-1.0.0-SNAPSHOT.jar /jdtls/java-analyzer-bundle/java-analyzer-bundle.core/target/
 COPY --from=fernflower /output/fernflower.jar /bin/fernflower.jar
 COPY --from=addon-build /app/hack/lsp-cli /bin/lsp-cli
+COPY --from=maven-index /maven.default.index /maven.default.index
```
Comment (shawn-hurley) [inline Dockerfile:36]: do we want this at the root of the container or should it be somewhere else?

**Review by shawn-hurley — APPROVED** [review]
Except for the root question, This looks good to me.

File: Dockerfile (hunk 1 above)
Comment (pranavgaikwad) [reply Dockerfile:36]: @shawn-hurley couldn't think of any other location, would `/usr/local/etc` be ok?

**Pull request merged**